src/screens/OtherProfile/index.js

```diff
@@ -642,7 +642,9 @@ const styles = StyleSheet.create({
   profileImage: {
     width: 95,
     height: 95,
-    borderRadius: 100
+    borderRadius: 100,
+    marginLeft: 3.2,
+    marginTop: 3.2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made to the `profileImage` style are fine, but it's important to note that the `marginLeft` and `marginTop` values are in pixels. If you want to support multiple screen sizes or densities, consider using a relative unit like percentage or `react-native`'s `Dimensions` API to calculate these values based on the screen size.

```diff
-    marginLeft: 3.2,
-    marginTop: 3.2
+    marginLeft: Dimensions.get('window').width * 0.01, // for example, 1% of screen width
+    marginTop: Dimensions.get('window').height * 0.01, // for example, 1% of screen height
```
This way, the margin will scale appropriately on different screen sizes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->